src/Lykke.Job.BlockchainCashoutProcessor/Wrokflow/Sagas/RiskControlSaga.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Lykke.Common.Chaos;
+using Lykke.Cqrs;
+using Lykke.Job.BlockchainCashoutProcessor.Contract;
+using Lykke.Job.BlockchainCashoutProcessor.Core.Domain.RiskControl;
+using Lykke.Job.BlockchainCashoutProcessor.Modules;
+using Lykke.Job.BlockchainCashoutProcessor.Wrokflow.Commands.CrossClient;
+using Lykke.Job.BlockchainCashoutProcessor.Wrokflow.Events.CrossClient;
+using Lykke.Job.BlockchainCashoutProcessor.Wrokflow.Events.RiskControl;
+using Lykke.Job.BlockchainRiskControl.Contract;
+using Lykke.Job.BlockchainRiskControl.Contract.Commands;
+using Lykke.Job.BlockchainRiskControl.Contract.Events;
+using Lykke.Job.BlockchainCashoutProcessor.Wrokflow.Commands.RiskControl;
+using Lykke.Job.BlockchainCashoutProcessor.Wrokflow.Commands.Regular;
+
+namespace Lykke.Job.BlockchainCashoutProcessor.Wrokflow.Sagas
+{
+    /// <summary>
+    /// -> Lykke.Job.TransactionsHandler : StartCashoutCommand
```

Comment:
It's Lykke.Service.Operations instead of Lykke.Job.TransactionsHandler at the moment